RobotsTxt/Drivers/RobotsTxtSettingsDisplayDriver.cs

```diff
@@ -37,7 +37,7 @@ public RobotsTxtSettingsDisplayDriver(IAuthorizationService authorizationService
 
         #region Overrides
 
-        public override async Task<IDisplayResult> EditAsync(RobotsTxtSettings settings, BuildEditorContext context)
+        public override async Task<IDisplayResult> EditAsync(RobotsTxtSettings section, BuildEditorContext context)
```

Comment:
Why the renaming of `settings` to `section`?

Reply:
`section` is the name of the parameter in the abstract method that we're overriding. SonarCloud will raise a warning if the override parameter names don't match.